src/main/java/uk/gov/hmcts/reform/unspec/model/Organisation.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.unspec.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@Builder
+@Jacksonized
+public class Organisation {
```

Comment:
Odd they decided to create this as an object, seems unnecessary 🤷‍♂️

Reply:
more details coming in the future? 🤔